test/rest.browser.test.js

```diff
@@ -300,6 +300,185 @@ describe('strong-remoting-rest', function(){
     });
   });
 
+    describe('Checks argument matches enum', function () {
+      describe('in top level', function () {
+        it(', should not allow parameters that are required and not found in enum', function(done) {
```

Comment:
I suggest replacing occurrences of `it(', should..` with `it('should..` because I don't see the `, ` convention used in any of the existing test code.